mitxgraders/__init__.py

```diff
@@ -25,7 +25,7 @@
 from mitxgraders.listgrader import *
 from mitxgraders.formulagrader import *
 from mitxgraders.sampling import *
-from mitxgraders.baseclasses import ConfigError, InvalidInput
+from mitxgraders.exceptions import ConfigError, StudentFacingError
 from mitxgraders.helpers import *
```

Comment:
I'm still concerned that `helpers` has nothing defined in it. Did you mean to expose a bunch of the math functions?